tests/driver_enc28j60/commands.c

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (C) 2022 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       Shell commands for netdev_eth test application
```

Comment:
Maybe this could be turned into a utility module? That would also allow using other applications to use this for debugging. I'm also personally not a fan of symlinking source files for code dedulication, as it makes it non-obvious that a seemingly application local file is relevant to other apps. (Also the `can_fast_ci_run.py` can be fooled by symlinks. But I guess this should be considered a bug of that tool.)

Reply:
> I'm also personally not a fan of symlinking source files for code dedulication, as it makes it non-obvious that a seemingly application local file is relevant to other apps.

I'm actually following the current way. We have a `tests/driver_netdev_common` which is used to test almost all network devices, via symbolic links. 

So we would have a module for Ethernet, one for IEEE 802.15.4, and so on?

Reply:
OK. Let's keep it consistent. (And maybe a future PR can convert them all to modules.)